### DISCUSSION TITLE
控制台报错：Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'ClockRange')     at makeRealTimePos (index.vue:374:5)

### DISCUSSION THREAD

**Discussion by coderyx-dev** [zouyaoji/vue-cesium#580] [Q&A]
第一次加载页面的时候可以正常显示，切换回来的时候就会提示该错误Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'ClockRange')，同时页面显示效果如下。
![微信图片_20240709150451](https://github.com/zouyaoji/vue-cesium/assets/85341021/cd3d7f83-02e8-4f31-a2b9-48656e7c496d)
请求一下解决办法。

**Comment by zouyaoji** [comment]
vc-viewer组件上绑定 :removeCesiumScript="false"
